docs/content/reference/routing-configuration/http/middlewares/passtlsclientcert.md

```diff
@@ -362,5 +259,5 @@ If there are more than one certificate, they are separated by a `,`.
 The following example shows such a concatenation, when all the available fields are selected:
 
 ```text
-Subject="DC=org,DC=cheese,C=FR,C=US,ST=Cheese org state,ST=Cheese com state,L=TOULOUSE,L=LYON,O=Cheese,O=Cheese 2,CN=*.example.com";Issuer="DC=org,DC=cheese,C=FR,C=US,ST=Signing State,ST=Signing State 2,L=TOULOUSE,L=LYON,O=Cheese,O=Cheese 2,CN=Simple Signing CA 2";NB="1544094616";NA="1607166616";SAN="*.example.org,*.example.net,*.example.com,test@example.org,test@example.net,10.0.1.0,10.0.1.2"
+Subject="DC=org,DC=cheese,C=FR,C=US,ST=Cheese org state,ST=Cheese com state,L=TOULOUSE,L=LYON,O=Cheese,O=Cheese 2,CN=*.example.com";Issuer="DC=org,DC=cheese,C=FR,C=US,ST=Signing State,ST=Signing State 2,L=TOULOUSE,L=LYON,O=Cheese,O=Cheese 2,CN=Simple Signing CA 2";NB="1747282426";NA="1778818426"SAN="*.example.org,*.example.net,*.example.com,test@example.org,test@example.net,10.0.1.0,10.0.1.2"
```

Comment:
```suggestion
Subject="DC=foo,DC=bar,C=foo,C=bar,ST=foo,ST=bar,L=foo,L=bar,O=foo,O=bar,CN=foo.bar";Issuer="DC=foo,DC=bar,C=foo,C=bar,ST=foo,ST=bar,L=foo,L=bar,O=foo,O=bar,CN=foo.bar";NB="1747282426";NA="1778818426"SAN="foo.com,bar.com"
```

I suggest also using neutral values, but I am not even sure this makes it clearer.
However, I think the current values doesn't help regarding understanding nor readability.
Maybe this is not the correct way to illustrate the concatenation.